.github/workflows/master.yml

```diff
@@ -27,8 +27,9 @@ jobs:
         run: go mod download
       - name: Build Binaries
```

Comment:
we remove this ones since we have a new workflow (ok to test)

Reply:
Don't we want to test master after we merged a PR? Technically, it's possible, that changes in a branch are ok, but break something after merged

Reply:
Don't we want to test master after we merged a PR? Technically, it's possible, that changes in a branch are ok, but break something after merged